### DISCUSSION TITLE
is it possible to update the ore Controll plugin?

### DISCUSSION THREAD

**Discussion by M0nkeyDMaik** [DerFrZocker/Ore-Control#40] [General]
update to 1.21.3 ?

**Comment by DerFrZocker** [comment]
Currently focused on other stuff, so will probably take some time for an official release, you can find however an untested dev build of version 1.21.3 (courtesy of @Joo200) here: https://jenkins.derfrzocker.de/job/Ore-Control-Dev/152/

**Reply by M0nkeyDMaik** [reply]
thx, but unfortunately it didnt work

**Reply by hensolo11** [reply]
oh just tested it too and it is not working for 1.21.3, the dev build works fine in 1.21.1 but not in 1.21.3
